kubectl-plugin/install/kk.go

```diff
@@ -103,7 +103,7 @@ func (o *kkOption) preRunE(cmd *cobra.Command, args []string) (err error) {
 		dep["crictl"] = "kubernetes-sigs/cri-tools"
 		dep["runc"] = "opencontainers/runc"
 	}
-	if err = is.CheckDepAndInstall(dep); err == nil {
+	if err = is.CheckDepAndInstall(dep); err == nil && o.container == "docker" {
```

Comment:
IMO, if the `container` isn't `docker`, then set default config files for containerd.

```diff
- err == nil && o.container == "docker"
+ err == nil && o.container != "docker"
```